crates/rpc/src/v02/method/estimate_fee.rs

```diff
@@ -253,7 +269,7 @@ pub(crate) mod tests {
 
             let storage = pathfinder_storage::Storage::migrate(db_path, JournalMode::WAL)
                 .unwrap()
-                .create_pool(NonZeroU32::new(1).unwrap())
+                .create_pool(NonZeroU32::new(10).unwrap())
```

Comment:
Where did this cause issues?

Reply:
In a previous state of the storage transaction for looking up the block_id wasn't immediately closed and was kept open during estimation. This shouldn't be an issue now.